k8_cortx_cloud/deploy-cortx-cloud.sh

```diff
@@ -1399,10 +1428,15 @@ deployCortxLocalBlockStorage
 deleteStaleAutoGenFolders
 deployCortxConfigMap
 deployCortxSecrets
-deployCortxControl
-deployCortxData
-deployCortxServer
-deployCortxHa
+
+if [ $DEPLOYMENT_MODE -eq $STANDARD_DEPLOYMENT ];  then
```

Comment:
Please change to switch case so that more cases can be added later.

Reply:
Fixed, added switch case and tested